src/js/utils/RouterUtil.js

```diff
@@ -186,6 +186,43 @@ const RouterUtil = {
       const blob = new Blob([data], { type: "application/json" });
       global.navigator.msSaveOrOpenBlob(blob, filename);
     }
+  },
+
+  /**
+   * Get URL full path. If it is a full path url return, if not attach
+   * the current local origin.
+   *
+   * @param {String} url The url
+   * @returns {String} - The url with the full path
+   */
+  getFullPath(url) {
```

Comment:
Why is this method needed? As far as I can tell, this is already handled by `Util.parseUrl` and I think all the tests will still pass without this method.

In parseUrl:
```
    const aElement = document.createElement("a");
    aElement.href = url;
```

Reply:
They don't pass because for the relative path the host is null. It adds the url to the href but it doesn't mean that it will add a host to it or a full path.